ai-platform-unified/notebooks/notebook_template.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
<p>Install additional dependencies -&gt; Install additional package dependencies</p><p> </p><p>in Notebook environment -&gt; in your notebook environment</p><p> </p><p>adanet -&gt; AdaNet</p><p> </p><p>tf-hub -&gt; TensorFlow Hub</p><p> </p><p>framework -&gt; package</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/251/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done</p>